plugin/trino-singlestore/src/test/java/io/trino/plugin/singlestore/TestSingleStoreConnectorTest.java

```diff
@@ -63,47 +63,28 @@ public final void destroy()
         singleStoreServer.close();
     }
 
-    @SuppressWarnings("DuplicateBranchesInSwitch")
     @Override
     protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
-        switch (connectorBehavior) {
-            case SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_EQUALITY:
-            case SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY:
-                return false;
-
-            case SUPPORTS_AGGREGATION_PUSHDOWN:
-                return false;
-
-            case SUPPORTS_JOIN_PUSHDOWN:
-                return true;
-            case SUPPORTS_JOIN_PUSHDOWN_WITH_FULL_JOIN:
-            case SUPPORTS_JOIN_PUSHDOWN_WITH_DISTINCT_FROM:
-                return false;
-
-            case SUPPORTS_RENAME_SCHEMA:
-                return false;
-
-            case SUPPORTS_CREATE_TABLE_WITH_TABLE_COMMENT:
-            case SUPPORTS_CREATE_TABLE_WITH_COLUMN_COMMENT:
-            case SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS:
-                return false;
-
-            case SUPPORTS_ADD_COLUMN_WITH_COMMENT:
-            case SUPPORTS_SET_COLUMN_TYPE:
-                return false;
-
-            case SUPPORTS_COMMENT_ON_TABLE:
-            case SUPPORTS_COMMENT_ON_COLUMN:
-                return false;
-
-            case SUPPORTS_ARRAY:
-            case SUPPORTS_ROW_TYPE:
-                return false;
-
-            default:
-                return super.hasBehavior(connectorBehavior);
-        }
+        return switch (connectorBehavior) {
+            case SUPPORTS_JOIN_PUSHDOWN -> true;
+            case SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_EQUALITY,
```

Comment:
this change makes it harder to see for example the fact that JOIN pushdown with full join or distinct from is not supported.

The logical separation across type of functionality is practically useful when reviewing documentation changes or debugging when users complain that something did not work because it's easy to skim the `hasBehaviour` method and see what is and is not supported.

Reply:
We can still use switch expressions but the logical separation needs to be retained.

Also I'd have appreciated if reviews were requested from maintainers who maintain this part of code.

Reply:
> this change makes it harder to see for example the fact that JOIN pushdown with full join or distinct from is not supported.

I strongly disagree with that.  I am a new reader of this codebase, and the original style is very hard to understand what is going on.
If you look at the last code in martin's comment above, I can quickly see:
* only `SUPPORTS_JOIN_PUSHDOWN` returns true
* all other cases return false
* the cases have enum values sorted alphabetically, so it is easy to search for the desired value.
* there is no default case, so the switch must covering

This makes the code much easier to navigate, and I think greatly helps readability over the orignal approach.  

The original approach has merit for someone attempting to review all such switches in Trino, but even in that case, the solution is brittle.  For the original code to be more efficient, the reader would need to know that the code consistent, and even then, they would need to verify that someone didn't mess up the ordering.  Additionally, the code above is consistently ordered alphabetically, so you still get the quick scan capabilities.

As an aside, the last item is very powerful as it means that new behaviors will cause a compile error until the case is adjusted.